src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -62,7 +62,7 @@ abstract class AbstractListBuilder implements ListBuilderInterface
     /**
      * The limit for this query.
      *
-     * @var int
+     * @var int|null
```

Comment:
Limit null means no limit. We handle it in the `DoctrineListBuilder` for all other use cases, I don't know. But since it's explicitly initialized with `null`, it should be handled.